### DISCUSSION TITLE
怎么换头像和名字啊？

### DISCUSSION THREAD

**Discussion by 17352696817** [share121/inter-knot#2981] [其他]
![Uploading image.png…]()

**Comment by cpu644** [comment]
您好！这里是绳网技术支援团队。在github右上角点击你的头像：
![image](https://github.com/user-attachments/assets/b78832c4-8a46-49c8-ba68-b56cf197f76c)
并点击your profile / 你的档案。然后点击你的头像（这次在正中间，而非右上角）
然后你就可以编辑你的名字/头像了！
<img width="680" alt="image" src="https://github.com/user-attachments/assets/9c21ae18-3b20-4069-a051-fbe1129f85fc">
头像图片最好为png/jpg并小于1mb
